docs/docs/self-host/customize-deployment/environment-variables.mdx

```diff
@@ -82,3 +82,11 @@ These variables enable you to control Single Sign On (SSO) functionality.
 | `LIGHTDASH_LOG_FILE_FORMAT`    | The format of log messages in the log file                  |           | `LIGHTDASH_LOG_FORMAT` |
 | `LIGHTDASH_LOG_FILE_PATH`      | The path to the log file                                    |           | `./logs/all.log`       |
 
+# dbt semantic layer environment variables
+
+| Variable                                     | Description                            | Required? | Default                                                         |
+|----------------------------------------------|----------------------------------------|-----------|-----------------------------------------------------------------|
+| `DBT_CLOUD_DOMAIN`                           | dbt cloud domain                       |           | https://semantic-layer.cloud.getdbt.com                         |
```

Comment:
We need to provide more context as dbt still doesn't have public docs on this.
And fix required and default column
Feel free to change the copy

```suggestion
| `DBT_CLOUD_DOMAIN`                           | dbt cloud Semantic Layer API domain           | Only required if your dbt cloud URL is similar to this `client123.us1.dbt.com`. If so, you should set the value `https://client123.semantic-layer.us1.dbt.com`     |    https://semantic-layer.cloud.getdbt.com                                                             |
```

Reply:
thanks!